packages/playground/apps/default/utils/editor.ts

```diff
@@ -121,8 +129,10 @@ export async function mountDefaultDocEditor(collection: DocCollection) {
               return Promise.resolve();
             },
           };
-          pageRootService.docModeService = mockDocModeService(
-            pageRootService.doc.id
+          pageRootService.std.container.override(
+            DocModeFeature,
+            MockDocModeService,
+            [BlockStdScope]
```

Comment:
Why does override need to explicitly declare the `[BlockStdScope]` it depends on?